lib/arcBase.js

```diff
@@ -30,8 +30,9 @@ var arc = module.exports = Base.extend({
         var chart = this.attributes.opts.chart;
         var data = this.attributes.data;
         var svg = this._make('svg',{
-            width: chart.width,
-            height: chart.height,
+            "style": "width: 100%; max-width: "
```

Comment:
Hi, note previous comment. The ```height=""``` and ```width=""``` attributes are not allowed on ```<div>``` elements. Also, in this case specifically ```max-width=""``` is not a valid attribute either.